scripts/deploy_oak_functions_loader

```diff
@@ -62,7 +62,7 @@ gcloud beta run deploy "${FUNCTIONS_INSTANCE_NAME}" \
 readonly HOST_NAME="$(gcloud api-gateway gateways describe "${GATEWAY_ID}" --location="${GCP_REGION}" --format='value(defaultHostname)')"
 
 #Build the client to test the connection.
-cargo build --manifest-path=./oak_functions/client/Cargo.toml
+./scripts/docker_run cargo build --manifest-path=./oak_functions/client/Cargo.toml
```

Comment:
And for this?

Reply:
We don't have a separate command for running clients. 
I could use `./scripts/runner --logs run-functions-examples --example-name=weather_lookup --run-server=false`. But I am not sure if it is possible to correctly pass `HOST_NAME` to the client. Also, do we want to get all the logs from the runner in the output?

Reply:
This is just the build step, can we run it with --run-server=false and --run-client=false, or something like that? Just to get the client binary built.